packages/renderer/src/lib/preferences/ProviderActionButtons.svelte

```diff
@@ -68,6 +68,14 @@ const showSetupButton = $derived(
 
 const showUpdateButton = $derived(provider.updateInfo?.version && provider.version !== provider.updateInfo?.version);
 
+const isDisabled = $derived(provider.kubernetesProviderConnectionCreation ? provider.status !== 'ready' : false);
+
+const disabledReason = $derived(
+  provider.kubernetesProviderConnectionCreation && provider.status !== 'ready'
+    ? (provider.warnings?.[0]?.details ?? 'Provider not ready')
+    : 'Provider unavailable',
+);
```

Comment:
```suggestion
const disabledReason = $derived(
  isDisabled ? (provider.warnings?.[0]?.details ?? 'Provider not ready') : `Create new ${providerDisplayName}`,
);
```

Reply:
disable reason should contain only the reason, but not 'Create new ...' label